### DISCUSSION TITLE
New Training Data - Enermess watermeter. analog ROI 0.1

### DISCUSSION THREAD

**Discussion by marc4s** [jomjol/AI-on-the-edge-device#1571] [General]
### Files

![Bildschirmfoto vom 2022-12-12 21-47-07](https://user-images.githubusercontent.com/93420756/207151294-7dde5056-03bb-4e34-998b-581409842e61.png)

I think I found the needle from the outer right analog ROI. but not sure. mine seems to be more bigger in the wrong direction which makes it often failing to see it correct.
currently it shows in the middle of 0 and 1 - so which would be 0,5 -> 1

as it is the 0,1 m³ it is to much away from the pre-value... any suggestions instead of break the glas and change the color of the pointy? hahaha...

**Comment by jomjol** [comment]
Hi @marc4s,
I don't understand you comment. All your needles should work, beside:
1) 0.001
2) Reflection on 0.01 is a problem.

You labeled "new training data". Where do you have it?

**Comment by marc4s** [comment]
forget the new training data. I just thought it might be here better then a bug.
I am trying a lot. so ignore the reflections on all below 0,1. the main problem for me is 0,1 currently

**Comment by marc4s** [comment]
why will 0.001 not work? I recognized this also, but today I found this needle in the training data...
is there a forum where to put those questions? sorry for bothering here in github

**Reply by caco3** [reply]
The discussions here are the right place.

**Comment by jomjol** [comment]
Sorry I meant 0.001.
You can make a discussion in this github instead of a issue. I'll convert it.
